src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -17886,5 +17886,49 @@ static void Main(string[] args)
             Assert.Equal(newText.ToString(), finalText);
             // no crash
         }
+
+        [Fact]
+        [WorkItem(13767, "https://github.com/dotnet/roslyn/issues/13767")]
+        public void TupleInConstant()
+        {
+            var source = @"
+class C<T> { }
+class C
+{
+    static void M()
+    {
+        const C<(int, int)> c = null;
+        System.Console.Write(c);
+    }
+}";
+
+            var comp = CreateCompilationWithMscorlib(source, assemblyName: "comp", references: s_valueTupleRefs);
+            comp.VerifyEmitDiagnostics();
```

Comment:
Please emit the binary and verify that the required type references are now emitted.